projects/rero/ng-core/src/lib/record/editor/type/textarea/textarea.component.ts

```diff
@@ -0,0 +1,138 @@
+/*
+ * RERO angular core
+ * Copyright (C) 2020 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import {
+  ChangeDetectionStrategy,
+  Component,
+  OnInit
+} from '@angular/core';
+import { AbstractControl, ValidatorFn } from '@angular/forms';
+import { FormlyFieldTextArea } from '@ngx-formly/bootstrap';
+
+@Component({
+  selector: 'ng-core-editor-formly-field-textarea',
+  template: `
+    <textarea
+      [formControl]="formControl"
+      [cols]="to.cols"
+      [rows]="to.rows"
+      class="form-control"
+      [class.is-invalid]="showError"
+      [formlyAttributes]="field"
+    >
+    >
+    </textarea
+    >
+    <ng-container
+      *ngIf="
+        field.templateOptions.limitWords || field.templateOptions.displayWords
+      "
+    >
+      <span class="small text-muted d-inline-block mr-3"
+        >{{ 'Number of words' | translate }}: {{ countWords }}
+        <ng-container *ngIf="field.templateOptions.limitWords">
+          / {{ field.templateOptions.limitWords }}</ng-container
+        ></span
+      >
+    </ng-container>
+    <ng-container
+      *ngIf="
+        field.templateOptions.limitChars || field.templateOptions.displayChars
+      "
+    >
+      <span class="small text-muted d-inline-block"
+        >{{ 'Number of chars' | translate }}: {{ countChars
+        }}<ng-container *ngIf="field.templateOptions.limitChars">
+          / {{ field.templateOptions.limitChars }}</ng-container
+        ></span
+      >
+    </ng-container>
```

Comment:
Why you don't use `ng-template` with a parameter `display` only with values `word` or `char`

Reply:
I'm not sure to understand, could you be more precise, please?

Reply:
OK, I changed the way to handle this.